extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -701,6 +702,30 @@ export class VisibilityManagerForDoc extends VisibilityManager {
       this.unsubscribe(
         this.observe(rootElement, this.setRootVisibility.bind(this))
       );
+      // Observe inabox window resize event.
+      const resizeListener = () => {
+        const id = getElementId(rootElement);
+        const trackedRoot = this.trackedElements_[id];
+        if (!trackedRoot) {
+          return;
+        }
+        if (
+          this.ampdoc.win./*OK*/ innerHeight < 1 ||
```

Comment:
any reason why not comparing to 0?

Reply:
Ohhh, i could compare it to 0 here. Unlike `getBoundingClientRect`, `innerHeight` returns an integer value. 
But It's the same.  I'll leave it as it is if there's no strong objection : )